python-backend/app/api/parties/party/resources/party_resource.py

```diff
@@ -148,8 +147,7 @@ def delete(self, party_guid):
         if party is None:
             return self.create_error_payload(404, 'Party guid with "{party_guid}" not found.'), 404
 
-        mine_party_appts = MinePartyAppointment.find_all_by_mine_party_appt_guid(
-            party_guid)
+        mine_party_appts = MinePartyAppointment.find_by_party_guid(party_guid)
```

Comment:
This was a bug that wasn't marking mine party appts as deleted

Reply:
Can you delete that method `find_all_by_mine_party_appt_guid` that filters by deleted_ind is misleading and it was only used here.